doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -12,6 +12,7 @@ The Uno Platform Hot Reload feature provides a way to modify the XAML and C# of
 - XAML and [C# Markup](xref:Uno.Extensions.Markup.Overview) Hot Reload for **iOS, Catalyst, Android, WebAssembly, and Skia (X11, Windows, macOS and FrameBuffer)**
 - All **[C# of Hot Reload](https://learn.microsoft.com/visualstudio/debugger/hot-reload)** in both Visual Studio and VS Code. See [supported code changes](https://learn.microsoft.com/visualstudio/debugger/supported-code-changes-csharp).
```

Comment:
@rajamatt you will need to fix some spell check and markdown linting issues:
![image](https://github.com/unoplatform/uno/assets/16295702/fa2ccef8-60ca-40b5-b332-528eeaa3c855)
- [Markdown linting errors](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=132025&view=logs&jobId=e61e0ee1-f3da-5962-52cf-48bdde3a5bd4&j=e61e0ee1-f3da-5962-52cf-48bdde3a5bd4&t=c18ecb7c-3c97-55c3-9bb9-5ef1777cbc25)
- [Spell check errors](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=132025&view=logs&jobId=050fb64b-9b12-52cc-b750-ec5d4e778b08&j=050fb64b-9b12-52cc-b750-ec5d4e778b08&t=669790bd-240a-5aea-d705-86a7326c36e7)

*@rajamatt I will suggest you to install the markdown lint extension for VS Code, helpful while writing documentation to test locally for markdown linting errors
https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint